app/views/bookings/_card.html.erb

```diff
@@ -23,16 +23,20 @@
             <p><strong>Renting ends: </strong> <%= booking.date_to %></p>
             <p><strong>Total days: </strong> <%=  pluralize( booking.days, 'Day' ) %></p>
             <p><strong>Total amount: </strong>¥ <%= number_to_currency(booking.full_price, unit: "", precision: 0) %></p>
-            <div class="btn-group-booking flex-sm-column">
+            <div class="btn-group-booking-landowner">
               <% if booking.status == 'pending' %>
-                <%= simple_form_for booking, url: landowner_booking_path(booking) do |f| %>
-                  <%= f.input :status, as: :hidden, input_html: { value: 'confirmed'} %>
-                  <%= f.submit 'Confirm booking', class: 'btn btn-primary booking-btn' %>
-                <% end %>
-                <%= simple_form_for booking, url: landowner_booking_path(booking) do |f| %>
-                  <%= f.input :status, as: :hidden, input_html: { value: 'rejected'} %>
-                  <%= f.submit 'Reject booking', class: 'btn btn-outline-danger booking-btn' %>
+                <div class="">
```

Comment:
Should this be empty? @joycehwchan

Reply:
This div doesn't need any classes. Gonna remove that, thanks.